.circleci/config.yml

```diff
@@ -478,7 +478,7 @@ jobs:
             export audius_ipfs_port=5001
             export audius_redis_url=redis://localhost:6379/0
             export audius_delegate_owner_wallet=0x1D9c77BcfBfa66D37390BF2335f0140979a6122B
-            export audius_delegate_private_key=0x3873ed01bfb13621f9301487cc61326580614a5b99f3c33cf39c6f9da3a19ca
+            export audius_delegate_private_key=0x3873ed01bfb13621f9301487cc61326580614a5b99f3c33cf39c6f9da3a19cad
```

Comment:
extra `d` at the end?

Reply:
intentional. this didn't work because it was missing the final letter. was very annoying to figure out

Reply:
im so sorry